src/i18n/locales/en/translation.json

```diff
@@ -0,0 +1,144 @@
+{
+  "welcome": {
```

Comment:
Eventually we will need to split this file up otherwise it will grow to big I'm worried. But no need to do it now.

Reply:
any thoughts on how to determine where to make the split? Should it be done by top-level section? (i.e. "welcome", "choose_connector", etc.)

Reply:
That could work. We don't need to decide now. Let's see how this grows and split after.